components/org.wso2.carbon.business.rules.web/src/components/auth/SecuredRouter.jsx

```diff
@@ -68,28 +63,39 @@ export default class SecuredRouter extends React.Component {
 
             const params = Qs.stringify({ referrer });
             return (
-                <Redirect to={{pathname: `${appContext}/login`, search: params}}/>
+                <Redirect to={{ pathname: `${appContext}/login`, search: params }} />
             );
         }
 
         return (
             <Switch>
-                <MuiThemeProvider theme={theme}>
-                    <div>
-                        <Redirect to={`${appContext}/businessRulesManager`}/>
-                        <Route exact path={`${appContext}/businessRuleCreator`} component={BusinessRuleCreator}/>
-                        <Route exact
-                               path={`${appContext}/businessRuleFromScratchForm/:formMode/templateGroup/:templateGroupUUID?/businessRule/:businessRuleUUID?`}
-                               component={BusinessRuleFromScratchForm}/>
-                        <Route exact
-                               path={`${appContext}/businessRuleFromTemplateForm/:formMode/templateGroup/:templateGroupUUID?/businessRule/:businessRuleUUID?`}
-                               component={BusinessRuleFromTemplateForm}/>
-                        <Route exact path={`${appContext}/businessRulesManager`} component={BusinessRulesManager}/>
-                        <Route exact path={`${appContext}/templateGroupSelector/:mode`}
-                               component={TemplateGroupSelector}/>
-                    </div>
-                </MuiThemeProvider>
+                <div>
+                    <Redirect to={`${appContext}/businessRulesManager`} />
+                    <Route exact path={`${appContext}/businessRuleCreator`} component={BusinessRuleCreator} />
+                    <Route
+                        exact
+                        path={`${appContext}/businessRuleFromScratchForm/:formMode/` +
+                        'templateGroup/:templateGroupUUID?/businessRule/:businessRuleUUID?'}
+                        component={BusinessRuleFromScratchForm}
+                    />
+                    <Route
+                        exact
+                        path={`${appContext}/businessRuleFromTemplateForm/:formMode/` +
+                        'templateGroup/:templateGroupUUID?/businessRule/:businessRuleUUID?'}
+                        component={BusinessRuleFromTemplateForm}
+                    />
+                    <Route exact path={`${appContext}/businessRulesManager`} component={BusinessRulesManager} />
+                    <Route
+                        exact
+                        path={`${appContext}/templateGroupSelector/:mode`}
+                        component={TemplateGroupSelector}
+                    />
+                </div>
```

Comment:
inside `switch` no need have a `div`.

Reply:
Components referred by `path`s are not rendering without the `div`. Therefore keeping it as it is.

Reply:
k